src/mongodb-migrations.coffee

```diff
@@ -176,6 +176,8 @@ class Migrator
         if err
           return cb err
         files = files
+          .filter (f) ->
+            path.extname(f) in ['.js', '.coffee'] && !f.startsWith('.')
```

Comment:
Can you please use `and not ` here? A matter of style, but consistent with the rest of the code.